src/frontend/js/components/SearchFilterGroupModal/_styles.scss

```diff
@@ -70,6 +70,7 @@
     background: r-theme-val(search-filters-group-modal, close-background);
     border: none;
     border-top: 1px solid r-theme-val(search-filters-group-modal, close-border);
+    margin-top: -1px; // Avoid having a double border when the button overlaps with a list item
   }
 
   &__overlay {
```

Comment:
Can you also increase the `z-index: 9999` because currently the modal is under the `topbar` element which has a `z-index: 200`...